pkg/apis/volumesnapshot/v1beta1/types.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package v1alpha1
+package v1beta1
 
 import (
 	core_v1 "k8s.io/api/core/v1"
```

Comment:
```GO
	// Optional: what happens to a snapshot content when released from its snapshot.
	// The default policy is Delete if not specified.		// The default policy is Delete if not specified.
	// +optional		// +optional
	DeletionPolicy *DeletionPolicy `json:"deletionPolicy,omitempty" protobuf:"bytes,4,opt,name=deletionPolicy"`		DeletionPolicy *DeletionPolicy `json:"deletionPolicy,omitempty" protobuf:"bytes,4,opt,name=deletionPolicy"`
```

StorageClass calls this `ReclaimPolicy`. Let's keep the same names, where we can, for consistency?

Reply:
P2